wagtailio/static/css/pages/services.scss

```diff
@@ -0,0 +1,362 @@
+/* ============================================
+
+  Services page
+
+*/
+.template-services-page {
+  $content-width: 800px;
+
+  $color--primary: $color--deep-indigo;
+  color: $color--lighter-indigo;
+
+  // Overrride for .page-content > .container a
+  .container .card .card__link,
+  .container a.link-button {
+    border: 0;
+  }
+
+  a,
+  a:hover {
+    h2,
+    h3 {
+      color: $color--primary;
+    }
+  }
+
+  .section {
```

Comment:
I'd usually expect to find template overrides within the elements partial, e.g.
```
.section {
    .template-services-page & {
        // ...overrides
     }
}
```
but can see that this project follows the format of how you've done overrides here, so that's good with me (consistency is king!)

Reply:
Yea, the overriding is getting a bit dirty in the project in general. Not sure why some styles like the anchor element need such a high specificity (`.page-content > .container a`). That makes it really hard to use anchor elements for anything. You always have to override the that style..